README.md

```diff
@@ -162,7 +162,174 @@ useradd --system --user-group --create-home myapp
 
 That should do it.
 
-###
+## Truecolor Over SSH With Wish
+
+### Noteworthy environment variables (for debugging)
+
+`TERM` - provides information about your terminal emulator's capabilities.
+
+`COLORTERM` - provides information about your terminal emulator's color
+capabilities. Used primarily to specify if your emulator has `truecolor`
+support.
+
+`NO_COLOR` - turns colors on and off. `NO_COLOR=1` for non-colored text
+outputs, `NO_COLOR=0` for colored text outputs.
+
+`CLICOLOR` - turns colors on and off. `CLICOLOR=1` for colored text outputs,
+`CLICOLOR=0` for non-colored text outputs.
+
+`CLICOLOR_FORCE` - overrides `CLICOLOR`.
+
+NO_COLOR vs CLICOLOR: if NO_COLOR is set or CLICOLOR=0 then the output should
+not be colored. Otherwise, the output can include ansi sequences.
+
+### Color profiles? What, like it's hard?
+
+In an SSH session, the client only sends the `TERM` environment variable, which
+can only detect `256 color` support. If you run `echo $COLORTERM` in your shell
+you'll likely see `truecolor` as the result, which is what you want for ultra
+colorful terminal outputs. If you don't see that as a result, it might be time
+to try a new [terminal emulator][supported-emulators]. 
```

Comment:
on another note, on termenv, we set truecolor for some terms anyway when we know they support it 

https://github.com/muesli/termenv/blob/345783024a348cbb893bf6f08f1d7ab79d2e22ff/termenv_unix.go#L53

Reply:
that's not on the latest termenv release though!

Reply:
Oh! I'll maybe leave a note that it's on the roadmap for termenv then 🤔 fwiw termstandard has a list of emulators that support truecolor, might be able to make that list even longer in termenv https://github.com/termstandard/colors?tab=readme-ov-file#fully-supporting I'll make an issue to that effect right meow